tests/ingest_runs/test_ingest_runs_mutations.py

```diff
@@ -142,3 +172,35 @@ def test_cancel_ingest_run(
         assert resp.json()["data"]["cancelIngestRun"]["ingestRun"] is not None
     else:
         assert resp.json()["errors"][0]["message"] == "Not allowed"
+
+
+@pytest.mark.parametrize(
+    "user_group",
+    [
+        ("Administrators"),
+    ],
+)
+def test_cancel_ingest_run_event(
+    db, clients, ingest_runs, mock_ingest_job, user_group
+):
+    """
+    Test that events are emitted correctly.
+    """
+    client = clients.get(user_group)
+
+    ingest_run = ingest_runs(n=1)[0]
+
+    resp = client.post(
+        "/graphql",
+        data={
+            "query": CANCEL_INGEST_RUN,
+            "variables": {
+                "id": to_global_id("IngestRunNode", ingest_run.id),
+            },
+        },
+        content_type="application/json",
+    )
+
+    assert Event.objects.count() == 1
```

Comment:
Same comment as above, except could you move this into the `test_cancel_ingest_run`